shared/featureconfig/config.go

```diff
@@ -50,6 +50,7 @@ type Flags struct {
 	DontPruneStateStartUp                      bool   // DontPruneStateStartUp disables pruning state upon beacon node start up.
 	NewStateMgmt                               bool   // NewStateMgmt enables the new experimental state mgmt service.
 	EnableInitSyncQueue                        bool   // EnableInitSyncQueue enables the new initial sync implementation.
+	EnableFieldTrie                            bool   // EnableFieldTrie enables the state from using field specific tries when computing the root.
```

Comment:
Where is this used?

Reply:
This is the flag we use to gate the feature in `state_trie.go`